src/mesh/X3D_Draco_Mesh_Reader.cc

```diff
@@ -34,9 +34,8 @@ X3D_Draco_Mesh_Reader::X3D_Draco_Mesh_Reader(
     : filename(filename_), bdy_filenames(bdy_filenames_),
       bdy_flags(bdy_flags_) {
   // check for valid file name
-  Insist(filename_.size() > 0, "No file name supplied.");
-  Insist(bdy_flags_.size() <= bdy_filenames_.size(),
-         "Number of B.C.s > number of boundary (side) node files.");
+  Require(filename_.size() > 0);
+  Require(bdy_flags_.size() <= bdy_filenames_.size());
```

Comment:
I'm not complaining, but why are the Insists changed to Requires?

Reply:
These are conditions that can and should be checked by the client.